http_client.go

```diff
@@ -10,6 +10,8 @@ import (
 
 var client = new(http.Client)
 
+type resourceResponse interface{}
```

Comment:
OIC. I guess that makes sense, although I wouldn't typecast it like here (just use `interface{}` directly in the method that does the work).